packages/core/src/components/icon/icon.tsx

```diff
@@ -87,7 +87,9 @@ export interface IIconProps extends IIntentProps, IProps {
 }
 
 @polyfill
-export class Icon extends AbstractPureComponent2<IIconProps & React.DOMAttributes<HTMLElement>> {
+export class Icon extends AbstractPureComponent2<
+    IIconProps & Omit<React.HTMLAttributes<HTMLElement>, "title"> & React.DOMAttributes<HTMLElement>
```

Comment:
we don't need `React.DOMAttributes` anymore, since `React.HTMLAttributes` extends it.

Reply:
Ah this is my bad, didn't see it in the first place.